packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue

```diff
@@ -49,6 +49,7 @@
         data-testid="page-header-actions"
       >
         <slot name="actions" />
+        <div id="action-button" />
```

Comment:
this is a pretty generic class - it may make sense to change this to something more specific, e.g. `kong-ui-app-page-header-action-button`

Reply:
Updated